src/plugins/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -238,7 +238,12 @@ export function ChangeDataView({
   return (
     <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
       <>
-        <EuiFlexItem grow={true}>
+        <EuiFlexItem
+          grow={true}
+          css={css`
+            min-width: 0;
+          `}
```

Comment:
We could also extract 

```
css`
            min-width: 0;
          `
```

into a variable outside of the component.

Reply:
Done ✅